Do not install include files and library itself when it's an
internal static library.
--- a/meson.build
+++ b/meson.build
@@ -77,7 +77,6 @@
   src,
   link_args: link_args,
   vs_module_defs: 'win32/zlib.def',
-  install: true,
   version: meson.project_version(),
 )
 
@@ -89,15 +88,4 @@
 )
 meson.override_dependency('zlib', zlib_dep)
 
-install_headers(headers)
-
-pkg = import('pkgconfig')
-
-pkg.generate(
-  zlib,
-  name: 'zlib',
-  description: 'zlib compression library',
-  extra_cflags: solo_args,
-)
-
 subdir('test')
